html/browsers/browsing-the-web/history-traversal/browsing_context_name_cross_origin_2.html

```diff
@@ -6,7 +6,7 @@
 <pre id="step_log"></pre>
 <iframe id="test"></iframe>
 <script>
-var t = async_test(undefined, {timeout:10000});
+var t = async_test(undefined);
```

Comment:
Can you also remove the undefined arguments? They were only there to make it possible to pass in the second argument.